### PR TITLE
SISRP-13264 - Build "Academic Dates..." card in delegate user's default view ("Toolbox")

### PR DESCRIPTION
**Pull request by dfkaye**
https://jira.berkeley.edu/browse/SISRP-13264

cc @johncrossman @christianvuerings

### CODE REVIEW

**Comment by johncrossman** [comment]
@dfkaye Dreaded conflicts must be resolved. Also, nit-picky request: please submit these changes in a fresh PR with only one commit. If you need to make adjustments to the PR you can use 
```
git commit --amend src/   
push -f 
```
In general, fewer commits makes release management and code reviews easier.

File: src/assets/javascripts/angular/controllers/widgets/academicDatesController.js [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    },
+    {
+      date: 'Jun 6',
+      title: 'Ten-Week Session begins'
+    },
+    {
+      date: 'Jun 20',
+      title: 'Eight-Week Session begins'
+    }
+  ];
```
Comment (johncrossman) [inline src/assets/javascripts/angular/controllers/widgets/academicDatesController.js]: @dfkaye it's nice to lay the groundwork for future API integration although I don't have a problem hardcoding all of this in `academic_dates.html`. We could then jettison `isLoading`. Perhaps @christianvuerings has an opinion...

File: src/assets/templates/toolbox.html [hunk 2]
```diff
@@ -13,6 +13,7 @@ <h1 class="cc-heading-page-title">My Toolbox</h1>
     </div>
     <div class="medium-4 columns cc-column-last" data-ng-if="api.user.profile.isSuperuser">
       <div data-ng-include="'widgets/toolbox/api_test.html'"></div>
+      <div data-ng-include="'widgets/toolbox/academic_dates.html'"></div>
```
Comment (johncrossman) [inline src/assets/templates/toolbox.html]: @dfkaye the include needs `data-ng-if="api.user.profile.isDelegateUser"`. Keep in mind, the toolbox page is also viewed by admins and advisors.

Comment (dfkaye) [inline src/assets/templates/toolbox.html]: @johncrossman does it also need `api.user.profile.features.csDelegatedAccess`?  I see that in other include directives...

Comment (johncrossman) [inline src/assets/templates/toolbox.html]: @dfkaye It does not. I believe this is the area of conflict. The latest code does not reference feature flag in the template. If feature flag is false then `isDelegateUser` will always be false.

**Comment by dfkaye** [comment]
Closing this PR - opening another from a patch branch shortly...

**Pull request closed without merging**